index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <nav class="menu">
+            <div class="nav-div"> Jhon Doe</div>
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+
+        </nav>
+    </header>
+    <main class="content">
+        <h1>Anibal Amoroso</h1>
+        <h2>Software Developer</h2>
+        <p class="text">I can help you build a product , feature or website Look through some of my work and experience! If
+            you like what
+            you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] The container of the section is a section tag (Please wrap this inside a section tag)